packages/react-native-bpk-component-phone-input/stories.js

```diff
@@ -224,23 +222,15 @@ class FullyIntegrated extends React.Component<
   }
 }
 
-storiesOf('react-native-bpk-component-phone-input/Integrated', module)
-  .add('Full component example', () => (
-    <FullyIntegrated initiallySelectedId="CA" codes={codes} />
-  ))
-  .add('Full component example with theme', () => (
-    <BpkThemeProvider theme={themeAttributes}>
-      <FullyIntegrated initiallySelectedId="CA" codes={codes} />
-    </BpkThemeProvider>
-  ));
```

Comment:
Think we still want these themed examples - it still does theming of other colours, just not grey

Reply:
This component doesn't seem to be using other theming so can't see anything changing